src/utils/sourcemaps.rs

```diff
@@ -11,17 +11,26 @@ use console::style;
 use indicatif::ProgressStyle;
 use log::{debug, info, warn};
 use sha1_smol::Digest;
+use symbolic::debuginfo::js::{
+    discover_debug_id, discover_sourcemap_embedded_debug_id, discover_sourcemaps_location,
+};
 use symbolic::debuginfo::sourcebundle::SourceFileType;
 use url::Url;
 
-use crate::api::Api;
+use crate::api::{Api, ChunkUploadCapability, NewRelease};
 use crate::utils::enc::decode_unknown_string;
 use crate::utils::file_search::ReleaseFileMatch;
-use crate::utils::file_upload::{ReleaseFile, ReleaseFileUpload, ReleaseFiles, UploadContext};
+use crate::utils::file_upload::{FileUpload, SourceFile, SourceFiles, UploadContext};
 use crate::utils::logging::is_quiet_mode;
 use crate::utils::progress::ProgressBar;
 
 fn is_likely_minified_js(code: &[u8]) -> bool {
+    // if we have a debug id or source maps location reference, this is a minified file
```

Comment:
If it has a reference or debug id, it should be considered minified. Not doing this misclassifies a lot of files that we upload which then in turn breaks the detection for the new behavior.

Reply:
Weeeeell *puts glasses on*.

It doesn't have to be minified, but it was definitely transformed in one way or another. `tsc` will produce source map reference, but you can still output non-minified, non-obfuscated, and fully readable code 🌝